docs/userGuide/contentAuthoring.md

```diff
@@ -385,6 +385,10 @@ While authoring your website, you may want to have your own CSS or Javascript fi
 - Author your `<style>` elements for CSS files and `<link>` elements for Javascript files using HTML as shown below.
     - Ensure that your URLs start from the root directory, by using <code>{<span></span>{baseUrl}}/</code> when you are referencing your files.
 
+- Use `<head-top>` and `<head-bottom>` tags to control where your content will be placed.
+  - You must enclose your content in at least one tag for it to be placed in the `<head>`.
+  - If the other tag is not needed, you may omit it in your head file.

```

Comment:
`  - It is not mandatory to use both tags, you may omit either one of them in your head file if it is unused.`